perlmod/Fink/PkgVersion.pm

```diff
@@ -4105,22 +4105,37 @@ EOF
 
 	my $has_kernel_dep;
 	my $deps = $self->get_depends(0, 0); # get runtime dependencies
+	my $predeps = $self->pkglist('Pre-Depends');
 
-	foreach (@$deps) {
+	foreach (@$predeps) {
 		foreach (@$_) {
 			$has_kernel_dep = 1 if /^\Q$kernel\E(\Z|\s|\()/;
 		}
 	}
+	if (not $has_kernel_dep) {
+		foreach (@$deps) {
+			foreach (@$_) {
+				$has_kernel_dep = 1 if /^\Q$kernel\E(\Z|\s|\()/;
+			}
+		}
+	}
+	# Move to Pre-Depends once bootstrapping can statisfy it.
+	# dpkg-bootstrap needs to be fvp aware for this to happen.
 	push @$deps, ["$kernel (>= $kernel_major_version-1)"] if not $has_kernel_dep;
 
+	$control .= "Pre-Depends: " . &lol2pkglist($predeps) . "\n";
```

Comment:
$predeps seems like it could be null at this point, so here we would get a Pre-Depends with a blank value. I assume this would be resolved once the kernel-dep is shifted from Depends to Pre-Depends (per a few lines previous).